package.json

```diff
@@ -33,10 +33,10 @@
     "zone.js": "^0.6.23"
   },
   "devDependencies": {
+    "@angular/cli": "^1.0.0",
     "@angular/compiler-cli": "2.4.0",
```

Comment:
after doing git rebase it is not showing any commits if anybody can help ?
![screenshot from 2017-04-13 15-22-48](https://cloud.githubusercontent.com/assets/17461330/25015077/e4d97efe-2097-11e7-9292-0e6bfeb15a6c.png)